### ISSUE TITLE
Loss is too big

### ISSUE THREAD

**Issue by XiXiRuPan** [slryou41/AnchorLoss#3]
Hi I troubled with loss is NAN(first epoch loss is 5021 ,it is too large).Can you give me some advices?
Thanks very much.

**Comment by slryou41** [comment]
What's the number of classes? I recommend using warm-up with cross-entropy for the first few epochs (setting ``AnchorLoss(warm_up == True)``) and feed the epoch number while training. 

I think it's because of the sum over classes at line 93, so if you still get the issue with the warm-up, then comment line 93 (``loss = loss.sum(1)``).

**Comment by dxnana** [comment]
Could you please upload the loss for human posture estimation?
